.circleci/config.yml

```diff
@@ -10,6 +10,75 @@ jobs:
           command: echo "Hello world!"
     working_directory: /usr/src/artman/
 
+  build_and_push_docker_image:
+    working_directory: /usr/src/artman
+    docker:
+      - image: docker:17.05.0-ce-git
+    steps:
+      - checkout
+      - setup_remote_docker
+      - run:
+          name: Finger crossed!
```

Comment:
nit: "finger" -> "fingers" :)

Reply:
Woops, thanks for making it a bit easier.